schedule/yast/msdos/msdos@svirt-xen-pv.yaml

```diff
@@ -1,35 +1,17 @@
 ---
-name:           msdos@svirt-xen-pv
-description:    >
+name: msdos@svirt-xen-pv
+description: >
   Test for installation on msdos partition table.
 vars:
   YUI_REST_API: 1
 schedule:
-  - installation/bootloader_start
-  - installation/setup_libyui
-  - installation/access_beta_distribution
-  - installation/product_selection/install_SLES
-  - installation/licensing/accept_license
-  - installation/registration/register_via_scc
-  - installation/module_registration/skip_module_registration
-  - installation/add_on_product/skip_install_addons
-  - installation/system_role/accept_selected_role_text_mode
-  - installation/partitioning/msdos_partition_table
-  - installation/clock_and_timezone/accept_timezone_configuration
-  - installation/authentication/use_same_password_for_root
-  - installation/authentication/default_user_simple_pwd
-  - installation/installation_settings/validate_ssh_service_enabled
-  - installation/installation_settings/open_ssh_port
-  - installation/installation_settings/validate_default_target
-  - installation/launch_installation
-  - installation/confirm_installation
-  - installation/performing_installation/perform_installation
-  - installation/performing_installation/stop_timeout_system_reboot_now
-  - installation/performing_installation/update_virsh_config_to_boot_from_hd
-  - installation/performing_installation/confirm_reboot
-  - installation/performing_installation/reconnect_after_reboot
-  - installation/first_boot
-  - console/validate_partition_table_via_parted
-  - console/validate_blockdevices
-  - console/validate_free_space
-  - console/validate_read_write
+  suggested_partitioning:
+    - installation/partitioning/msdos_partition_table
+  default_systemd_target:
+    - installation/installation_settings/validate_default_target
+  installation_logs: []
```

Comment:
I think there is not reason to not do this, probably we missed in some point, it is good we do this iteration to notice.